app/lib/efile/nc/d400_calculator.rb

```diff
@@ -3,7 +3,19 @@ module Nc
     class D400Calculator < ::Efile::TaxCalculator
       attr_reader :lines
 
+      def initialize(year:, intake:, include_source: false)
+        super
+        @d400_schedule_s = Efile::Nc::D400ScheduleSCalculator.new(
+          value_access_tracker: @value_access_tracker,
+          lines: @lines,
+          intake: @intake
+        )
+      end
+
+
       def calculate
+        @d400_schedule_s.calculate
```

Comment:
does it not matter here that we're filling in line 27 & 41 first b/c it's just adding it to a hash & not to the xml yet in this `calculate` method?

Reply:
we're filling it in first because it determines line 9 of the d400 form